### PR TITLE
Fix RSS item URLs

### PR DESCRIPTION
**Pull request by kaushikp13**

### CODE REVIEW

File: src/utils/rss/generate-feed.js [hunk 1]
```diff
@@ -55,8 +58,11 @@ const generateItem = function generateItem(post) {
 const generateRSSFeed = function generateRSSFeed(siteConfig) {
   return {
     serialize: ({ query: { allGhostPost } }) =>
-      allGhostPost.edges.map((edge) =>
+      allGhostPost.edges.map((edge) =>{
+        console.log(edge);
         Object.assign({}, generateItem(edge.node))
```
Comment (IsAmrish) [inline src/utils/rss/generate-feed.js:63]: Pass `siteConfig` to `generateItem` function as second parameter to use it later.

File: src/utils/rss/generate-feed.js [hunk 2]
```diff
@@ -1,9 +1,12 @@
 const cheerio = require(`cheerio`)
 const tagsHelper = require(`@tryghost/helpers`).tags
 const _ = require(`lodash`)
+const config = require(`../../config/siteConfig`)
+
 
 const generateItem = function generateItem(post) {
```
Comment (IsAmrish) [inline src/utils/rss/generate-feed.js:7]: here => 

const generateItem = function generateItem(post, siteConfig)

File: src/utils/rss/generate-feed.js [hunk 3]
```diff
@@ -1,9 +1,12 @@
...
   const itemUrl = post.canonical_url || post.url
+  const itemFinalUrl= itemUrl.replace("https://admin.mayadata.io/", `${config.siteUrl}/blog/`);
```
Comment (IsAmrish) [inline src/utils/rss/generate-feed.js:9]: remove line 8 and 9

and use this instead

```
const itemUrl = `${siteConfig.siteUrl}/blog/${post.slug}/`;
```

File: src/utils/rss/generate-feed.js [hunk 4]
```diff
@@ -1,9 +1,12 @@
 const cheerio = require(`cheerio`)
 const tagsHelper = require(`@tryghost/helpers`).tags
 const _ = require(`lodash`)
+const config = require(`../../config/siteConfig`)
```
Comment (IsAmrish) [inline src/utils/rss/generate-feed.js:4]: remove this

**Review by IsAmrish — CHANGES_REQUESTED** [review]

File: src/utils/rss/generate-feed.js (hunk 1 above)
Comment (IsAmrish) [reply src/utils/rss/generate-feed.js:63]: remove console.log

File: src/utils/rss/generate-feed.js (hunk 3 above)
Comment (kaushikp13) [reply src/utils/rss/generate-feed.js:9]: image

those two things are different @IsAmrish still I need to make a link as you told me above?

Comment (IsAmrish) [reply src/utils/rss/generate-feed.js:9]: You are comparing two different objects.

**Review by IsAmrish — APPROVED** [review]
LGTM

**Pull request merged**